lib/store/src/types.ts

```diff
@@ -108,15 +108,15 @@ export type DocsIndexEntry = BaseIndexEntry & {
 };
 
 export type IndexEntry = StoryIndexEntry | DocsIndexEntry;
-export interface V2CompatIndexEntry extends StoryIndexEntry {
+export interface V2CompatIndexEntry extends Omit<StoryIndexEntry, 'type'> {
   kind: StoryIndexEntry['title'];
   story: StoryIndexEntry['name'];
-  parameters: Parameters;
+  parameters?: Parameters;
```

Comment:
@tmeasday why is this optional?

Reply:
In `stories.json` (statically generated version) we don't have any parameters, I guess? We could put `fileName` in there maybe?

Reply:
Makes sense. I just wanted to know the reasoning.